rdkit/Chem/Pharm3D/EmbedLib.py

```diff
@@ -746,7 +768,7 @@ def _getFeatDict(mol, featFactory, features):
   molFeats = {}
   for feat in features:
     family = feat.GetFamily()
-    if not family in molFeats:
+    if family not in molFeats:
```

Comment:
me-- for ever putting this horrible (though correct) construct into the code. Thanks for fixing it. Did the linter catch that?